src/ui/components/Background/Background.tsx

```diff
@@ -50,11 +50,11 @@ const bgClassNames = {
   transparent: 'transparent-bg',
 } as const;
 
-export function useBackgroundKind({
-  kind,
-}: {
-  kind: 'neutral' | 'white' | 'transparent';
-}) {
+type Props = { kind: 'neutral' | 'white' | 'transparent' };
+
+export const whiteBackgroundKind: Props = { kind: 'white' };
```

Comment:
why not exporting just Props as a type?

Reply:
The idea is to export a stable object so that you don't need to wrap in useMemo for popular cases:

```
useBackgroundKind(useMemo(() => ({ kind: 'white' }), []))
```
vs
```
useBackgroundKind(whiteBackgroundKind)
```